scss/components/_buttons.scss

```diff
@@ -2,8 +2,7 @@ $btn-border-width: 4px;
 
 @mixin disabled-btn-before {
   &::before {
-    box-shadow:
-      0 $btn-border-width rgba($white, 0.4) inset,
+    box-shadow: 0 $btn-border-width rgba($white, 0.4) inset,
```

Comment:
This codebase doesn't use prettier.  Can you revert the code style changes?

Reply:
I will revert this